moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -70,37 +63,22 @@ export default function MapView({ projectName, projectStatuses }) {
   const mapRef = useRef();
   const { projectId } = useParams();
 
+  /* sets the type of geometry to use in component edit mode. allowed values
+  are `points`, `lines`, or `null` */
+  const [linkMode, setLinkMode] = useState(null);
+
   /* tracks a component clicked from the list or the projectFeature popup */
   const [clickedComponent, setClickedComponent] = useState(null);
 
   /* tracks a projectFeature clicked from the map */
   const [clickedProjectFeature, setClickedProjectFeature] = useState(null);
 
-  /* holds the state of a component that's being created */
-  const [draftComponent, setDraftComponent] = useState(null);
-
   /* tracks a projectFeature hovered on map */
   const [hoveredOnMapFeature, setHoveredOnMapFeature] = useState(null);
 
-  /* sets the type of geometry to use in component edit mode. allowed values
-  are `points`, `lines`, or `null` */
-  const [linkMode, setLinkMode] = useState(null);
-
-  /* if a component is being edited */
-  const [isEditingComponent, setIsEditingComponent] = useState(false);
-
-  /* if a component is being deleted */
-  const [isDeletingComponent, setIsDeletingComponent] = useState(false);
-
   /* tracks the loading state of AGOL feature service fetching */
   const [isFetchingFeatures, setIsFetchingFeatures] = useState(false);
 
-  const [showComponentEditDialog, setShowComponentEditDialog] = useState(false);
-
-  const [showEditModeDialog, setShowEditModeDialog] = useState(false);
-
-  const [addProjectComponent] = useMutation(ADD_PROJECT_COMPONENT);
-  const [deleteProjectComponent] = useMutation(DELETE_PROJECT_COMPONENT);
```

Comment:
All of these were moved into either `useCreateComponent`, `useUpdateComponent`, or `useDelete` except of `isFeatchingFeatures`